website/static/js/folderPickerNodeConfig.js

```diff
@@ -427,6 +430,24 @@ var FolderPickerViewModel = oop.defclass({
                 self.loading(false);
                 // Set flag to prevent repeated requests
                 self.loadedFolders(true);
+            },
```

Comment:
The only reason behind adding this in parent was the assumption that file names(for any addon) with special characters would be resolved. I'm not sure if any other addons allows special characters for filenames. If not allowed, then I'll move this to oauthAddonNodeConfig